manifests/config/bootstrap.pp

```diff
@@ -19,15 +20,20 @@
     }
   }
 
-  if versioncmp($::puppetversion, '4.0.0') >= 0 {
-    $targetfile = '/etc/puppetlabs/puppetserver/bootstrap.cfg'
+  if $targetfile {
+    $targetfile_real = $targetfile
   } else {
-    $targetfile = '/etc/puppetserver/bootstrap.cfg'
+    if versioncmp($::puppetversion, '4.0.0') >= 0 {
```

Comment:
shouldn't this be using the version of the puppetserver software and not the agent?

Reply:
Approving this though, since it maintains the current functionality.

Reply:
Yes that's exactly what I described in the issue https://github.com/voxpupuli/puppet-puppetserver/issues/52. From my knowledge there is no puppetserver version fact available to achieve that.

We had the same issue in your ssh module [1] and I found no workaround for puppet master related version numbers.

[1] https://github.com/ghoneycutt/puppet-module-common/blob/master/manifests/mkuser.pp#L113-L117

Reply:
Could you add a small comment to the code and link to your issue? That would really help those who might be debugging this issue as it is possible they have the new agent and slightly older puppetserver.

Reply:
comment added